RecoBTag/DeepFlavour/interface/c_pf_features_converter.h

```diff
@@ -0,0 +1,263 @@
+#ifndef RecoSV_DeepFlavour_c_pf_features_converter_h
+#define RecoSV_DeepFlavour_c_pf_features_converter_h
+
+#include "RecoBTag/DeepFlavour/interface/deep_helpers.h"
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
+
+namespace deep {
+
+  // conversion map from quality flags used in PV association and miniAOD one
+  constexpr int qualityMap[8]  = {1,0,1,1,4,4,5,6};
+
+  enum qualityFlagsShiftsAndMasks {
```

Comment:
enum class maybe?

Reply:
@slava77 regarding the enum vs enum class I would prefer to keep enum for consistency with https://github.com/cms-sw/cmssw/blob/f45ea8cea768ce0790b5961898dc9d1bd6dd9318/DataFormats/PatCandidates/interface/PackedCandidate.h#L760
but I will put them both in the scope of  the new ChargedCandidateConverter class.

Reply:
I put them in the scope in https://github.com/cms-sw/cmssw/pull/19893/commits/c58a7db3f5b57532c429a7f2f102b462285b57a4, did not change to enum class though